src/Components/HelpBubbleIcon/HelpButton.tsx

```diff
@@ -0,0 +1,37 @@
+import { IconButton } from '@mui/material';
+import { useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { HelpBubbleIcon } from './helpBubbleIcon.tsx';
+
+const HelpButton = ({
```

Comment:
No need to change, but this component can't be used if the parent has `display: flex`.